modules/Bitfinex2Poloniex.py

```diff
@@ -0,0 +1,114 @@
+
+    @staticmethod
+    def convertOpenLoanOffers(bfxOffers):
+        '''
+        Convert from "offers" to "returnOpenLoanOffers"
+        '''
+        plxOffers = {}
+        for offer in bfxOffers:
+            if offer['currency'] not in plxOffers:
+                plxOffers[offer['currency']] = []
+
+            if offer['direction'] == 'lend' and offer['remaining_amount'] != '0.0':
+                plxOffers[offer['currency']].append({
+                    "id": offer['id'],
+                    "rate": str(float(offer['rate'])/36500),
+                    "amount": offer['remaining_amount'],
+                    "duration": offer['period'],
+                    "autoRenew": 0,
+                    "date": Bitfinex2Poloniex.convertTimestamp(offer['timestamp'])
+                })
+
+        return plxOffers
+
+    @staticmethod
+    def convertActiveLoans(bfxOffers):
+        '''
+        Convert from "credits" to "returnActivLoans"
+        '''
+
+        plxOffers = {}
+        plxOffers['provided'] = []
+        plxOffers['used'] = []
+        for offer in bfxOffers:
+            plxOffers['provided'].append({
+                "id": offer['id'],
+                "currency": offer['currency'],
+                "rate": str(float(offer['rate'])/36500),
+                "amount": offer['amount'],
+                "duration": offer['period'],
+                "autoRenew": 0,
+                "date": Bitfinex2Poloniex.convertTimestamp(offer['timestamp'])
+            })
+
+        return plxOffers
+
+    @staticmethod
+    def convertLoanOrders(bfxLendbook):
+        '''
+        Converts from 'lendbook' to 'returnLoanOrders'
+        '''
+
+        plxOrders = {}
+        plxOrders['offers'] = []
+        plxOrders['demands'] = []
+
+        for bid in bfxLendbook['bids']:
+            plxOrders['demands'].append({
+                'rate': '{0:0.8f}'.format(float(bid['rate'])/36500),
+                'amount': bid['amount'],
+                'rangeMin': '2',
+                'rangeMax': bid['period']
+            })
+
+        for ask in bfxLendbook['asks']:
+            plxOrders['offers'].append({
+                'rate': '{0:0.8f}'.format(float(ask['rate'])/36500),
+                'amount': ask['amount'],
+                'rangeMin': '2',
+                'rangeMax': ask['period']
+            })
+
+        return plxOrders
+
+    @staticmethod
+    def convertAccountBalances(bfxBalances, account=''):
+        '''
+        Converts from 'balances' to 'returnAvailableAccountBalances'
+        '''
+        balances = {}
+
+        accountMap = {
+            'trading': 'margin',
+            'deposit': 'lending',
+            'exchange': 'exchange'
+        }
+
+        if (account == ''):
+            balances = {'margin': {}, 'lending': {}, 'exchange': {}}
+        else:
+            balances[account] = {}
+
+        for balance in bfxBalances:
+            if ((account == '' or account == accountMap[balance['type']])
```

Comment:
Apologies. I've always done it the other way and thought that was correct. I see in the actual PEP-8 they say it was like that for decades but has changed. 

Every day's a school day :)